source/exec.tex

```diff
@@ -5672,3 +5690,845 @@
+\linebreak scheduler_type(sch.scheduler))), *this);} % avoid Overfull
+\end{itemdescr}
+
+\begin{itemdecl}
+void uncaught_exception();
+\end{itemdecl}
+
+\begin{itemdescr}
+\pnum
+\effects
+If the signature \tcode{set_error_t(exception_ptr)} is not an element
+of \tcode{error_types}, calls \tcode{terminate()} \iref{except.terminate}.
+Otherwise, stores \tcode{current_exception()} into \exposid{errors}.
+\end{itemdescr}
+
+\begin{itemdecl}
+coroutine_handle<> unhandled_stopped();
+\end{itemdecl}
+
+\begin{itemdescr}
+\pnum
+\effects
+Completes the asynchronous operation associated with \tcode{STATE(*this)}
+by invoking \tcode{set_stopped(std::move(RCVR(*this)))}.
+\end{itemdescr}
+
+\begin{itemdescr}
+\pnum
+\returns
+\tcode{noop_coroutine()}.
+\end{itemdescr}
+
+\begin{itemdecl}
+@\unspec@ get_env() const noexcept;
+\end{itemdecl}
+
+\begin{itemdescr}
+\pnum
+\returns
+An object \tcode{env} such that queries are forwarded as follows:
+
+\begin{itemize}
+\item \tcode{env.query(get_scheduler)} returns \tcode{scheduler_type(SCHED(*this))}.
+\item \tcode{env.query(get_allocator)} returns \exposid{alloc}.
+\item \tcode{env.query(get_stop_token)} returns \exposid{token}.
+\item For any other query \tcode{q} and arguments \tcode{a...} a
+    call to \tcode{env.query(q, a...)} returns
+    \tcode{STATE(*this).} \tcode{environment.query(q, a...)} if this expression
+    is well-formed and \tcode{forwarding_query(q)} is well-formed and is \tcode{true}.  Otherwise
+    \tcode{env.query(q, a...)} is ill-formed.
+\end{itemize}
+\end{itemdescr}
+
+\begin{itemdecl}
+template<class... Args>
+void* operator new(size_t size, const Args&... args);
+\end{itemdecl}
+
+\begin{itemdescr}
+\pnum
+If there is no parameter with type \tcode{allocator_arg_t} then let
+\tcode{alloc} be \tcode{Allocator()}.  Let \tcode{arg_next} be the
+parameter following the first \tcode{allocator_arg_t} parameter (if
+any) and let \tcode{alloc} be \tcode{Allocator(arg_next)}.  Then
+\tcode{PAlloc} is \tcode{allocator_traits<Allocator>::template
+rebind_alloc\linebreak<U>} where \tcode{U} is an unspecified type % avoid Overfull
+whose size and alignment are both \tcode{__STDCPP_DEFAULT_NEW_ALIGNMENT__}.
+
+\pnum
+\mandates
+
+\begin{itemize}
+\item The first parameter of type \tcode{allocator_arg_t} (if any) is not the last parameter.
+\item \tcode{Allocator(arg_next)} is a valid expression if there is a parameter
+   of type \tcode{allocator_arg_t}.
+\item \tcode{allocator_traits<PAlloc>::pointer} is a pointer type.
+\end{itemize}
+
+\pnum
+\effects
+Initializes an allocator \tcode{palloc} of type \tcode{PAlloc} with
+\tcode{alloc}. Uses \tcode{palloc} to allocate storage for the
+smallest array of \tcode{U} sufficient to provide storage for a
+coroutine state of size \tcode{size}, and unspecified additional
+state necessary to ensure that \tcode{operator delete} can later
+deallocate this memory block with an allocator equal to \tcode{palloc}.
+\pnum
```

Comment:
```suggestion

\pnum
```

Reply:
Yes, pnum needs an empty line in front (or a \begin{itemdescr}, otherwise it's not functioning properly.